notebooks/llm-agent-langchain/llm-agent-langchain.ipynb

```diff
@@ -29,7 +29,9 @@
     "- [Create agent](#Create-agent)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    "gradio"\
</pre><p>"gradio&gt;=4.19" - there is no chatbot element in old versions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2083/discussion/#comment-1630739666'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>